lib/test_suite.py

```diff
@@ -90,7 +102,7 @@ def __init__(self, suite_path, args):
             raise RuntimeError("Suite %s doesn't exist" % repr(suite_path))
 
         # read the suite config
-        config = ConfigParser.ConfigParser()
+        config = configparser.ConfigParser()
```

Comment:
Added both options to ConfigParser() when running under Python 3 and updated commit message.

Reply:
I'll cite the new code:

```python
        # read the suite config
        packer_kwargs = dict()
        packer_kwargs['inline_comment_prefixes='] = ';' 
        packer_kwargs['strict='] = True
        try:
            # Python 3.2+
            config = configparser.ConfigParser(**packer_kwargs)
        except TypeError:
            # Python 2
            config = configparser.ConfigParser()
```

It is equivalent to `config = configparser.ConfigParser()`. Details:

1. `packer_` prefix is copy-pasted from the tarantool-python code, where keyword arguments for an msgpack packer instance keyword arguments. Here it has no sense.
2. `'='` is not part of an option name. There is no `inline_comment_prefixes=` option. That is why you got the `TypeError` even on Python 3.
3. According to the documentation the `inline_comment_prefixes` should be a tuple, e.g. `(';',)`.
4. You don't need to do any exception handling. Just add the Python 3 options into the dictionary when you're on Python 3. This is why the dictionary is needed: to pass options conditionally.

I have a little suggestion, which helps me to avoid most of mistakes of this kind. I often test code snippets in REPL when I'm in doubt regarding precise option names/types/format and so on. For example I catched the problem with multiline options (I mentioned it in the previous comment) this way.

Reply:
Thanks for comments. Fixed 1, 2, 3, 4.:

```
--- a/lib/test_suite.py
+++ b/lib/test_suite.py
@@ -97,15 +97,10 @@ class TestSuite:
             raise RuntimeError("Suite %s doesn't exist" % repr(suite_path))
 
         # read the suite config
-        packer_kwargs = dict()
-        packer_kwargs['inline_comment_prefixes='] = ';'
-        packer_kwargs['strict='] = True
-        try:
-            # Python 3.2+
-            config = configparser.ConfigParser(**packer_kwargs)
-        except TypeError:
-            # Python 2
-            config = configparser.ConfigParser()
+        parser_kwargs = dict()
+        parser_kwargs['inline_comment_prefixes'] = (';',)
+        parser_kwargs['strict'] = True
+        config = configparser.ConfigParser(**parser_kwargs)
         config.read(os.path.join(suite_path, "suite.ini"))
         self.ini.update(dict(config.items("default")))
         self.ini.update(self.args.__dict__)
```